src/Services/Identity/Identity.API/Identity.API.csproj

```diff
@@ -28,7 +28,6 @@
     <PackageReference Include="Microsoft.AspNetCore.HealthChecks" Version="1.0.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration.AzureKeyVault" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.AzureAppServices" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="1.0.172" />
```

Comment:
Can't remove building the client-side libraries, because they are not included in version control (as usual) so there's no guarantee that they'll be there when building the project.

For example, you have them in your computer because you built the project some time ago, but this would work if yu fresh cloned the project and tried building the project with Docker.

You'd have to build it first in VS, and that's nonsense when working with Docker, so that's why the building from LibMan is important.